### PR TITLE
Add Java EE 8 Security example

### PR DESCRIPTION
**Pull request by TomasHofman**
Example of secured servlet via Java EE 8 Security annotations.

### CODE REVIEW

File: security/ee-security-basic/pom.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  <artifactId>example-ee-security-basic</artifactId>
+
+  <name>Thorntail Examples: Java EE 8 Security - Basic Authentication</name>
+  <description>Thorntail Examples: Java EE 8 Security - Basic Authentication</description>
+
+  <packaging>war</packaging>
+
+  <properties>
+    <version.resteasy>3.0.14.Final</version.resteasy>
```
Comment (Ladicek) [inline security/ee-security-basic/pom.xml]: This should probably be 3.6.2.Final these days.

**Comment by Ladicek** [comment]
add to whitelist

File: security/ee-security-basic/pom.xml (hunk 1 above)
Comment (TomasHofman) [reply security/ee-security-basic/pom.xml]: Fixed.

File: security/ee-security-basic/pom.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  ~
+  ~ Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.thorntail.examples</groupId>
+    <artifactId>examples-jaxrs</artifactId>
```
Comment (Ladicek) [inline security/ee-security-basic/pom.xml]: This is probably what the build fails on -- should be `examples-security`.

Comment (TomasHofman) [reply security/ee-security-basic/pom.xml]: Fixed too.

Is it possible to get read access to CI server? I can't access the build info when anonymous and when I log in I can access nothing at all.

Comment (Ladicek) [reply security/ee-security-basic/pom.xml]: I tried to figure out why build logs are no longer accessible to anonymous users, but couldn't find anything :-( I'll take a look again. In the meantime, I tried to enable all authenticated users to have read permissions. Can you try now?

Comment (TomasHofman) [reply security/ee-security-basic/pom.xml]: Now it behaves the same logged in or not. Trying to access e.g. https://ci.wildfly-swarm.io/view/Examples/job/thorntail-examples-pull-request-linux/ or a particular build gives 404.

**Comment by Ladicek** [comment]
retest this please

**Comment by Ladicek** [comment]
Previously, the Windows jobs failed because they couldn't reach the Sonatype Snapshots repo. I configured the Windows agents to also use our local Nexus, so the Windows jobs should be passing now.

**Comment by TomasHofman** [comment]
Perfect, thanks for relaying the problems to me :). What about the Windows UberJar one, is there something relevant?

**Comment by Ladicek** [comment]
The Windows uberjar failure is unrelated (JSF test).

**Pull request merged**